src/screens/SurveyOverviewScreen.tsx

```diff
@@ -0,0 +1,83 @@
+import { useNavigation } from '@react-navigation/core';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { useCallback } from 'react';
+import { useQuery } from 'react-apollo';
+import { SectionList } from 'react-native';
+
+import { SectionHeader, TextListItem } from '../components';
+import { LoadingSpinner } from '../components/LoadingSpinner';
+import { texts } from '../config';
+import { combineLanguages } from '../helpers';
+import { usePullToRefetch, useSurveyLanguages } from '../hooks';
+import { SURVEYS } from '../queries/survey';
+import { Survey } from '../types';
+
+const useSurveySections = () => {
+  const { data: surveys, loading, refetch } = useQuery<{
+    ongoing: Survey[];
+    archived: Survey[];
+  }>(SURVEYS, { fetchPolicy: 'cache-and-network' });
```

Comment:
is the fetch policy safe? we cannot test changing a survey title for example, but i am not sure, if this would be recognized.
for "default" graphql queries (`graphqlFetchPolicy` in `graphqlHelper`) we use `network-only` if no cache refresh logic kicks in (`cache-first`)

Reply:
we will see, once we have count manipulation